Part_2_/Sorting.cs

```diff
@@ -16,6 +16,8 @@ public static T[] Order<T>(T[] collection)
         {
             // NOTE: do not use LINQ
             // TODO: implement here
+            Array.Sort(collection);
```

Comment:
And please split your changes: one PR for one Task

Reply:
remove this changes from this branch and after create another branch and implement it there